### PR TITLE
Provide better description for var.origin_bucket

### PR DESCRIPTION
**Pull request by nitrocode**
## what
* Provide better description for var.origin_bucket
* Provide an example of reusing an s3 bucket in the README

## why
* There's some confusion whether to define this value as the bucket to be created or to reuse an existing bucket

## references
* Closes https://github.com/cloudposse/terraform-aws-cloudfront-s3-cdn/issues/90

### CODE REVIEW

File: variables.tf [hunk 1]
```diff
@@ -49,7 +49,7 @@ variable "override_origin_bucket_policy" {
 variable "origin_bucket" {
   type        = string
   default     = ""
-  description = "Origin S3 bucket name"
+  description = "Use an existing origin S3 bucket name"
```
Comment (osterman) [inline variables.tf:52]: Name of an existing S3 bucket to use as the origin.

Comment (nitrocode) [reply variables.tf]: Good comment. I updated the description with that and more.

**Review by osterman — DISMISSED** [review]

**Review by jamengual — APPROVED** [review]

**Pull request merged**